src/lib.rs

```diff
@@ -37,7 +37,7 @@ use lightning::onion_message::offers::{OffersMessage, OffersMessageHandler};
 use lightning::routing::gossip::NetworkGraph;
```

Comment:
super nit: Change beginning of commit message to "Whenever an offer onion message arrives"